api/src/main/resources/ehcache.xml

```diff
@@ -9,7 +9,7 @@
     graphic logo is a trademark of OpenMRS Inc.
 
 -->
-<ehcache>
+<ehcache name="hibernateCache">
```

Comment:
I did it for debugging purposes, because default name is "_DEFAULT_". At first I had problem with those multiple CacheManager instances, so having those named was handy, and I think it can be useful in future if anybody will have to do something with those.